lib/battle/CObstacleInstance.h

```diff
@@ -120,6 +128,12 @@ struct DLL_LINKAGE SpellCreatedObstacle : CObstacleInstance
 		h & trap;
 
 		h & customSize;
+
+		if(version >= 821)

```

Comment:
You've bumped minimal format. No need to check for it - all saves with this code are at 821 or newer.

Reply:
done